files/user_resource_usage.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Script from https://git.ncsa.illinois.edu/ici-monitoring/ici-developed-checks/-/blob/main/login_nodes/login_node_resource_check.sh
+
+tfile=$(mktemp /tmp/user_resource.XXXXXXXXX)
+users=($(ps axo user:20 | sort -u | grep -v USER))
+ps axo user:20,pid,pcpu,pmem,rssize > "${tfile}"
+
+for u in ${users[@]}
+do
+	cpu_percent=$(egrep ^${u} ${tfile} | awk '{SUM+=$3}END{print SUM}')
+	mem_percent=$(egrep ^${u} ${tfile} | awk '{SUM+=$4}END{print SUM}')
+	mem_kb=$(egrep ^${u} ${tfile} | awk '{ sum+=$5} END {print int(sum)}')
+	num_processes=$(awk -v user=${u} '$1 == user {print $0}' ${tfile} | wc -l)
```

Comment:
I'm not sure why JD did that one differently. Technically the `awk -v user...` method was more precise since it explicitly looks for matches in the first column, making it immune to the other bug you found of `bsper` matching `bsper3`. But if it was to avoid that bug why didn't he update the other lines?

Now that the greps use `-w` though I don't see much point in having that more complicated awk. Ideally the searches for user should be the same, so flipped it to using `egrep -w` like the others.